Functions/xASL_quant_MultiPLD.m

```diff
@@ -263,21 +221,19 @@
 %% 6    Apply quantification
 CBF = PWI.*ScaleImage;
 
-if bUseBasilQuantification
-    CBF = xASL_stat_MeanNan(CBF, 4);
-end
+CBF = xASL_stat_MeanNan(CBF, 4);
 
 %% 6    Print parameters used
 fprintf('%s\n',' model with parameters:');
 
 if x.Q.ApplyQuantification(3)
     switch lower(x.Q.LabelingType)
         case 'pasl'
-            fprintf('%s',['TI1 = ' num2str(x.Q.LabelingDuration) ' ms, ']);
-            fprintf('%s',['TI (ms) = ' num2str(x.Q.Initial_PLD')]);  % transpose for multiPLD
+            fprintf('%s\n',['TI1 = ' num2str(unique(x.Q.LabelingDuration(:)')) ' ms,']);
```

Comment:
Perhaps nice to add to `xASL_num2str` that it will always do the `(:)'`.

Reply:
I have tested `xASL_num2str` and it doesn't do the (:)' automatically. 

`xASL_num2str([1;2;3])

ans =

  3×1 char array

    '1'
    '2'
    '3'`

Reply:
I meant to add this feature to xASL_num2str as we probably want this in all cases. Read my comment, I said "add to `xASL_num2str`, I didn't say "add `xASL_num2str`" :)

Reply:
New issue made for this. OK to resolve?
https://github.com/ExploreASL/ExploreASL/issues/995